core/templates/App.ts

```diff
@@ -79,6 +79,32 @@ angular.module('oppia').config([
   function(
       $compileProvider, $cookiesProvider, $httpProvider,
       $interpolateProvider, $locationProvider, $provide) {
+    var ugs = new UpgradedServices();
```

Comment:
Does this mean UpgradedServices will never go away?

Reply:
This will go away after we complete our migration to Angular, we won't need to use $provide.value for any service then.